sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3143,7 +3188,7 @@ object GpuOverrides {
       "The backend for hash based aggregations supporting TypedImperativeAggregate functions",
       ExecChecks(
         (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64 +
-          TypeSig.MAP + TypeSig.ARRAY + TypeSig.STRUCT)
+          TypeSig.MAP + TypeSig.ARRAY.nested(TypeSig.STRUCT + TypeSig.DOUBLE) + TypeSig.STRUCT)
```

Comment:
This is confusing because anything nested in a TypeSig applies to all nest-able types in the TypeSig.  Could you just update the whole thing to the following?

```
(TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64 +
  TypeSig.MAP + TypeSig.ARRAY + TypeSig.STRUCT).nested()
```